### PR TITLE
component-list-item-cy-209

### PR DESCRIPTION
**Pull request by nicolasshiken**
Figma:
Captura de Pantalla 2021-12-20 a la(s) 12 37 51

Render:
Captura de Pantalla 2021-12-20 a la(s) 12 38 16
Captura de Pantalla 2021-12-20 a la(s) 12 37 07
Captura de Pantalla 2021-12-20 a la(s) 12 37 10
Captura de Pantalla 2021-12-20 a la(s) 12 37 15
Captura de Pantalla 2021-12-20 a la(s) 12 37 19

### CODE REVIEW

File: src/components/component-image/ComponentImage.tsx [hunk 1]
```diff
@@ -1,8 +1,8 @@
-import React, { ReactNode } from 'react';
-import { SComponentImage } from './styles/SComponentImage';
-import { SInnerRectangle } from './styles/SInnerRectangle';
+import React, { ReactNode } from "react";
+import { SComponentImage } from "./styles/SComponentImage";
+import { SInnerRectangle } from "./styles/SInnerRectangle";
 
-export type IComponentImageTypes = "open" | "grid" | "list item";
+export type IComponentImageTypes = "open" | "grid" | "list-item";
```
Comment (diegomart2000) [inline src/components/component-image/ComponentImage.tsx:5]: @nicolasshiken you have changed this type but not the story.

File: src/icons/IconTags.tsx [hunk 2]
```diff
@@ -5,67 +5,219 @@ import { IIconSize } from "../utils/types";
 const IconSize = {
   12: (
     <SIconBase width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
-      <path d="M6.10985 6.7636C6.2054 6.85915 6.2054 7.01406 6.10985 7.10961C6.01431 7.20516 5.85939 7.20516 5.76384 7.10961C5.6683 7.01406 5.6683 6.85915 5.76384 6.7636C5.85939 6.66805 6.01431 6.66805 6.10985 6.7636" strokeWidth="1.2" strokeLinecap="round" strokeLinejoin="round"/>
-      <path fillRule="evenodd" clipRule="evenodd" d="M7.80667 4.19333L12.6133 9C13.128 9.51467 13.128 10.3493 12.6133 10.864L9.864 13.6133C9.34933 14.128 8.51467 14.128 8 13.6133L3.19333 8.80667C3.07 8.68333 3 8.51533 3 8.34067V4.65933C3 4.29533 3.29533 4 3.65933 4H7.34133C7.516 4 7.68333 4.06933 7.80667 4.19333V4.19333Z" strokeWidth="1.2" strokeLinecap="round" strokeLinejoin="round"/>
-      <path d="M13 6.66667L8.66267 2.38467C8.41267 2.138 8.07667 2 7.726 2H5" strokeWidth="1.2" strokeLinecap="round" strokeLinejoin="round"/>
+      <path
```
Comment (diegomart2000) [inline src/icons/IconTags.tsx:8]: @nicolasshiken please avoid these kind of updates that introduces noise to reviews and may cause conflicts with other PRs. This change is not related to the implementation.

**Review by diegomart2000 — CHANGES_REQUESTED** [review]

**Comment by diegomart2000** [comment]
@nicolasshiken also note that if the list of tags is larger than the teste one, the component render breaks.
Screen Shot 2021-12-21 at 6 30 59 PM

Image is narrower than the original.

File: src/components/component-image/ComponentImage.tsx (hunk 1 above)
Comment (nicolasshiken) [reply src/components/component-image/ComponentImage.tsx:5]: The story used `list-item` already, and the styledComponents use `list-item` as well, that's why I changed the interface. Unless I misunderstood?

File: src/icons/IconTags.tsx (hunk 2 above)
Comment (nicolasshiken) [reply src/icons/IconTags.tsx:8]: My bad! I must have changed something and then undone the change, but left the formatting there. I'll be more careful and discard the changes instead next time

Comment (nicolasshiken) [reply src/icons/IconTags.tsx:8]: My bad! I must have changed something and then undone the change, but left the formatting there. I'll be more careful and discard the changes instead next time

File: src/components/component-image/ComponentImage.tsx (hunk 1 above)
Comment (nicolasshiken) [reply src/components/component-image/ComponentImage.tsx:5]: The story uses `list-item` already, and the styledComponents use `list-item` as well, that's why I changed the type. Or did I misunderstood something?

File: src/icons/IconTags.tsx (hunk 2 above)
Comment (nicolasshiken) [reply src/icons/IconTags.tsx:8]: My bad! I must have changed something and then undone the change, but left the formatting there. I'll be more careful and discard the changes instead next time

**Review by diegomart2000 — APPROVED** [review]
